tests/backup/backup_helper.go

```diff
@@ -3910,6 +3911,50 @@ func CreateRestoreWithProjectMapping(restoreName string, backupName string, name
 	return nil
 }
 
+// CreateRestoreOnRancherWithoutCheck creates restore with project mapping
+func CreateRestoreOnRancherWithoutCheck(restoreName string, backupName string, namespaceMapping map[string]string, clusterName string,
+	orgID string, ctx context.Context, storageClassMapping map[string]string, rancherProjectMapping map[string]string, rancherProjectNameMapping map[string]string, replacePolicy ReplacePolicy_Type) error {
+
+	var bkp *api.BackupObject
+	var bkpUid string
+	backupDriver := Inst().Backup
+	log.Infof("Getting the UID of the backup %s needed to be restored", backupName)
+	bkpEnumerateReq := &api.BackupEnumerateRequest{
+		OrgId: orgID}
+	curBackups, err := backupDriver.EnumerateBackup(ctx, bkpEnumerateReq)
+	if err != nil {
+		return err
+	}
+	for _, bkp = range curBackups.GetBackups() {
+		if bkp.Name == backupName {
+			bkpUid = bkp.Uid
+			break
+		}
+	}
```

Comment:
How about using Inst().Backup.GetBackupUID?

Reply:
What value will it add?
@kphalgun-px

Reply:
Nothing. Only for the sake of brevity.